tech/guides/admin_management.md

```diff
@@ -0,0 +1,53 @@
+## Add/Remove admin
+Imagine, TokenD's user data has grown, Alice's process of review user's requests would take a while. In order to reduce Alice's effort and time, she could share her resposibilities to assistant Bob.

```

Comment:
Let's suppose Alice is the admin of the TokenD system. The project is growing and the process of review user's requests taking a while

Reply:
And let's use Charlie and Delta instead of Alice and Bob cause we have already reserved this names for users.